stylus/ui-app/notifications.styl

```diff
@@ -0,0 +1,54 @@
+/*!
+ * NOTIFICATIONS
+ * =============
+ *
+ * This file contains all needs for notifications
+ */
+
+
+$notifications
+    notification-width = (546/basefont)em
+    .coz-notifier
+        z-index 100
```

Comment:
Well I agree, we should secure the position of notifications on top of anything but by setting its z-index to 100 (on the premise that we used z-index incrementation by 1), it means that we allow 100 layers of positioned elements on a page. That is a lot. 
That's just my opinion but if we have a UI with 100 positioned elements, we really messed up the CSS and/or UI, big time. I'd say over 20 positioned elements it's already too much. 
Maybe we could at least go at 50 but I still think it would be too much.

But we could use the [postcss z-index plugin](https://github.com/ben-eb/postcss-zindex) that unify and simplify z-indexes all over CSS in a project ( poke @m4dz ). I used it before, it works fine. This way you can put the z-index at 50 or 100 if you like and the plugin would parse all the CSS and give the smallest number possible and still be the highest of the CSS file.